src/docs/development/tools/formatting.md

```diff
@@ -17,11 +17,11 @@ where time might be better spent on code behavior rather than code style.
 
 Install the `Dart` plugin (see [Editor setup](/docs/get-started/editor))
 to get automatic formatting of code in Android Studio and IntelliJ.
-
-To automatically format the code in the current source code window,
-right-click in the code window and select `Reformat Code with dartfmt`.
-You can add a keyboard shortcut to this in Keymap section of IntelliJ
-Preferences.
+To automatically format your code in the current source code window,
+use `Cmd+Alt+L` (on Mac) or `Ctrl+Alt+L` (on Windows and Linux).
+Android Studio and IntelliJ also provides a check box named `Format code on save` on
+the Flutter page in Preferences (on Mac) or Settings (on Windows and Linux)
+which will format the current file automatically when you save it.
```

Comment:
There's no Save File action in Android Studio/IntelliJ (well, strictly speaking it's there but its a kind of hidden action and has a different name). Typical action (Ctrl+S / Cmd+S) is Save All. So it would be more correct to write smth like `which will ensure that the IDE formats edited files when it saves them`. There might be better words, I'm not a native speaker.

Reply:
The option is defined as 'Format code on save' and there is no context of me referring to any action to save a file. Rather, it just conveys to the client that whenever they intend to save their files, it will be formatted.

Reply:
I mean words "will format the current file". There's no *current* file when we talk about saving files in the IDE. It formats *unsaved*, i.e. *edited* files, not *current*.

Reply:
> I mean words "will format the current file". There's no _current_ file when we talk about saving files in the IDE. It saves and formats _unsaved_, i.e. _edited_ files, not _current_.

Oh yes, you're right. Thanks for pointing that out!